src/code/audio/code_8019AF00.c

```diff
@@ -3477,17 +3621,46 @@ void AudioOcarina_ResetStaffs(void) {
     D_801D8534 = 0;
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_8019DFF8.s")
+void Audio_Noop1(s32 arg0) {
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_8019E000.s")
+void Audio_Noop2(s32 arg0, s32 arg1) {
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_8019AF00/func_8019E00C.s")
+void Audio_Noop3(void) {
+}
```

Comment:
Three of the 5 noop's line up with OoT's Audio Debug functions, so I'm presuming that's where they come from. Another 2 are also noop'd in OoT. It might even be worth naming three of the functions `AudioDebug_Noop1` - `AudioDebug_Noop3` but idk, it's hard to say